StartHere.md

```diff
@@ -20,22 +20,17 @@ Discuss issues you've faced, discover new & better ways of working (together), c
 
-## Suggest / Vote for Meetups
-
-<img src="images/github.png" align="right"  height="80" >
-
-We found GitHub to be very convenient for organizing our topics and voting for meetups. 
-
-If you are new to GitHub, don't worry. This is a great chance to start using it! 
+## Topics selection / Suggest
 
-Our topic voting method is simple, every quarter, we send a topic poll, on our [MeetUp Page](https://www.meetup.com/devstaff/), containing 5 or more topics. There, you rate all topics from 1 to 10, based on how much you prefer each topic to be in the next three. When the poll closes, the top three posts that have speakers are selected, and then presented in the next 3 meetups. This is a rule of thumb: changes can be made by the Org Team when required, including, but not limited to, opportunities for collaboration with other MeetUps, expert speakers in Crete, etc. But how does the Org Team select which topics will make it into the poll? It's simple: it looks in the Issues page of this project, and finds proposed issues with speakers. So if you want to propose a topic for the vote, go ahead and open a new Issue with the topic tag! If you can be a speaker in that, or any other topic, add a :speaker: (`:speaker:`) comment, so we know you are available to make a short presentation on this topic, along with other members of our community! There are not many requirements, and it's not something that you commit to for life. If you feel like you have anything to say, don't be afraid, add the emoji, and when this topic comes to a vote, we'll contact you, to see if you're still up for it, and can make a presentation about it in our meetup!
+The Org team is responsible for curating topics and finding speakers for the upcoming meetups. Any member of the community is free to suggest new topics and also propose himself/herself for being the next meetup speaker on his/her favorite topic, by filling this form [here](https://docs.google.com/forms/d/e/1FAIpQLSd-xubuaYZIl0kl1B5wqy_nIE-Rq010OfUEfAdMyoSA6S9jAg/viewform). The DevStaff Slack channel is also a great place to propose new topics to the Org team as well as pointing to speakers/topics that might be of interest to the community. So if you want to propose a topic, or you are available to make a presentation on a topic, go ahead and fill in the [form](https://docs.google.com/forms/d/e/1FAIpQLSd-xubuaYZIl0kl1B5wqy_nIE-Rq010OfUEfAdMyoSA6S9jAg/viewform) or head out to our Slack channel and drop us a line there! When there's interest in a certain topic the Org team actively tries to find more speakers willing to talk about it. There's always someone sharing the same passion with you!
```

Comment:
If we have a short URL for the form I would advise using it here as it keeps some statistics.